package/system/uci/Makefile

```diff
@@ -21,10 +21,14 @@ PKG_LICENSE:=LGPL-2.1
 PKG_LICENSE_FILES:=
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
+PKG_BUILD_DEPENDS := uci/host
```

Comment:
I'm not sure, whether this is correct. I don't know if this could potentially happen, but I foresee following issue: 

Imagine target package which doesn't depend on uci, but is using (or would use) host uci for uci-defaults fiddling, but it would fail, because host uci won't be available yet. Perhaps this dependency needs to be assured via `tools/Makefile` (which would probably mean, that the uci has to be part of the tools as well)?